src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -295,7 +295,20 @@ public static bool TryDecimal(FormattingInfo formatInfo, IRContext irContext, Fo
 
         // https://docs.microsoft.com/en-us/powerapps/maker/canvas-apps/functions/function-text
         public static FormulaValue Text(EvalVisitor runner, EvalVisitorContext context, IRContext irContext, FormulaValue[] args)
-        {
+        {
+            if (args.Length == 1 && (args[0] is BlankValue || (args[0] is UntypedObjectValue untypedObjectValue && untypedObjectValue.Impl.Type == FormulaType.Blank)))

```

Comment:
>args[0] is BlankValue || (args[0] is UntypedObjectValue untypedObjectValue && untypedObjectValue.Impl.Type == FormulaType.Blank [](http://example.com/codeflow?start=37&length=127)

Do we have other places that need this check?

I wonder if we should encapsulate it in a new helper like:
FormulaValue.IsBlank()

Reply:
Yes, we use in few places so I updated Greg's isblank function to share.